File: src/3190_find_minimum_operations_divisible_by_three/FindMinimumOperationsDivisibleByThree.java
public class FindMinimumOperationsDivisibleByThree {
    public int minimumOperations(int[] nums) {
        int ans = 0;

        for (int num : nums) {
            if (num % 3 != 0) {
                ans += 1;
            }
        }

        return ans;
    }

    public static void main(String[] args) {
        FindMinimumOperationsDivisibleByThree s = new FindMinimumOperationsDivisibleByThree();
        int[] nums1 = { 1, 2, 3, 4 };
        System.out.println(s.minimumOperations(nums1) + " 3");
        int[] nums2 = { 3, 6, 9 };
        System.out.println(s.minimumOperations(nums2) + " 0");
    }
}

File: src/3190_find_minimum_operations_divisible_by_three/find_minimum_operations_divisible_by_three.cpp
#include <iostream>
#include <vector>
using namespace std;

class Solution {
   public:
    int minimumOperations(vector<int>& nums) {
        int ans = 0;

        for (int num : nums) {
            if (num % 3 != 0) {
                ans += 1;
            }
        }

        return ans;
    }
};

int main() {
    Solution s;
    vector<int> nums1 = {1, 2, 3, 4};
    cout << s.minimumOperations(nums1) << " 3" << endl;
    vector<int> nums2 = {3, 6, 9};
    cout << s.minimumOperations(nums2) << " 0" << endl;
    return 0;
}
